sdk/go/src/sawtooth_sdk/messaging/stream.go

```diff
@@ -136,14 +169,16 @@ func (self *Stream) start() {
 	fmt.Println("Reactor exited:", err)
 }
 
-// Send messages from the outgoing channel in a loop
+// Handle a single message sendRequest
 func (self *Stream) sender(i interface{}) error {
+	// 1. Validate this is a sendRequest. Should never happen.
 	req, ok := i.(*sendRequest)
 	if !ok {
 		fmt.Println("Received unexpected type from channel!")
 		return nil
 	}
 
+	// 2. Deserialize
```

Comment:
Probably can just lose the numbers.  It has the natural ordering, and then it doesn't have to be updated if future steps are added in the middle.